presto-docs/src/main/sphinx/sql/select.rst

```diff
@@ -1062,6 +1062,12 @@ standard rules for nulls. The subquery must produce exactly one column::
     SELECT name
     FROM nation
     WHERE regionkey IN (SELECT regionkey FROM region)
```

Comment:
That is also a bad example query btw. It returns the same as 

Select name from nation

A better example could be 

select name from nation where regionkey in (select regionkey from region where name='AMERICA');

Reply:
Good point, I never paid much attention to the logic